src/glang/codegen/builtin_types.py

```diff
@@ -89,11 +89,10 @@ def get_name(self) -> str:
         return f"{self.name}{specialization}"
 
     def format_for_output_to_user(self, full=False) -> str:
-        reference_suffix = (
-            "&"
-            if self.is_reference and not self.definition.is_always_a_reference
-            else ""
-        )
+        if self.definition.is_always_a_reference:
+            reference_suffix = "" if self.is_reference else "unborrowed"
```

Comment:
Shouldn't this also be `" (unborrowed)"` or similar?

Reply:
I've fixed this in #126. We now print `u8[<unborrowed>]` in the same situation.